components/prism-diff.js

```diff
@@ -39,11 +39,28 @@
 			alias.push("bold");
 		}
 
-		Prism.languages.diff[name] = {
-			// pattern: /^(?:[_].*(?:\r\n?|\n|(?![\s\S])))+/m
-			pattern: RegExp('^(?:[' + prefix + '].*(?:\r\n?|\n|(?![\\s\\S])))+', 'm'),
-			alias: alias
-		};
+		var pattern = RegExp('^(?:[' + prefix + '].*(?:\r\n?|\n|(?![\\s\\S])))+', 'm');
+
+		if (['-', '<', '+', '>', '!'].indexOf(prefix) !== -1) {
```

Comment:
Why is the `if` still here?

Reply:
Updated. Misunderstood the `unchanged` one.